cmd/kube-proxy/app/server_others.go

```diff
@@ -329,7 +328,6 @@ func newProxyServer(
 		if err != nil {
 			return nil, fmt.Errorf("unable to create proxier: %v", err)
 		}
-		proxymetrics.RegisterMetrics()
```

Comment:
this was kind of weird, no? initializing in two different places instead doing just in one, maybe a copy paste thing